packages/strapi-plugin-upload/models/File.settings.json

```diff
@@ -80,7 +80,8 @@
     "related": {
       "collection": "*",
       "filter": "field",
-      "configurable": false
+      "configurable": false,
+      "autoPopulate": false
```

Comment:
I think we should rather remove the default auto populate for the medialib fetch all in the controllers so we don't break the api responses

Reply:
It seems that controllers use service call which is just plain call to `strapi.query()`. Not sure I got right understanding of your comment

Reply:
At https://github.com/strapi/strapi/blob/55eaa91fec2fffb77be644f686b36c2400191425/packages/strapi-plugin-upload/controllers/upload/admin.js#L40 you could do 

```js
const files = await strapi.plugins.upload.services.upload[method](query, []);
```

and in the service change

```js

  fetchAll(params) {
    combineFilters(params);
    return strapi.query('file', 'upload').find(params);
  },

  search(params) {
    return strapi.query('file', 'upload').search(params);
  },


```

to 

```js

  fetchAll(params, populate) {
    combineFilters(params);
    return strapi.query('file', 'upload').find(params, populate);
  },

  search(params, populate) {
    return strapi.query('file', 'upload').search(params, populate);
  },

```

Reply:
I think you could event update the controller findOne in the same logic so the load of one media in the media lib would be faster too

Reply:
Ok, got it. Will change

Reply:
Sources update